packages/account-v2/src/mocks/idv-form.mock.ts

```diff
@@ -0,0 +1,124 @@
+import { ResidenceList } from '@deriv/api-types';
+
+export const DOCUMENT_LIST = [
+    {
+        id: 'aadhaar',
+        text: 'Aadhaar Card',
+        additional: {
+            display_name: 'PAN Card',
+            format: '^[a-zA-Z]{5}\\d{4}[a-zA-Z]{1}$',
+            example_format: 'ABCDE1234F',
+        },
+        value: '^[0-9]{12}$',
+        example_format: '123456789012',
+    },
+    {
+        id: 'drivers_license',
+        text: 'Drivers License',
+        value: '^[a-zA-Z0-9]{10,17}$',
+        example_format: 'AB1234567890123',
+    },
+    {
+        id: 'epic',
+        text: 'Voter ID',
+        value: '^[a-zA-Z]{3}[0-9]{7}$',
+        example_format: 'ABC1234567',
+    },
+    {
+        id: 'pan',
+        text: 'PAN Card',
+        value: '^[a-zA-Z]{5}\\d{4}[a-zA-Z]{1}$',
+        example_format: 'ABCDE1234F',
+    },
+    {
+        id: 'passport',
+        text: 'Passport',
+        additional: {
+            display_name: 'File Number',
+            format: '^.{15}$',
+            example_format: 'AB1234567890123',
+        },
+        value: '^.{8}$',
+        example_format: 'A1234567',
+    },
+];
+
+export const SELECTED_COUNTRY: ResidenceList[0] = {
```

Comment:
Same here

Reply:
I think the convention for using configs is Upper Snake case if I'm not wrong